web/cobrands/sass/_base.scss

```diff
@@ -2499,8 +2499,16 @@ label .muted {
 }
 
 a#geolocate_link {
-  display: inline-block;
-  padding: 0.5em;
+  display: inline-block !important;
```

Comment:
Would be nice to have a comment on this line, explaining why we’re using an `!important` – what selector is it overriding? Makes it easier to refactor stuff later, if we ever want to.